lib/prompt.ml

```diff
@@ -21,14 +21,9 @@ let rec loop ~question ~default_answer =
              for the default");
       loop ~question ~default_answer
 
-let confirm ~question ~yes =
-  if yes then true else loop ~question ~default_answer:Yes
+let confirm ~question ~yes ~default_answer =
+  if yes then true else loop ~question ~default_answer
```

Comment:
I would make `default_answer` optional and make the default behavior to `Yes` since it's the main usecase (and adding the argument everywhere generates an unnecessary diff)

Reply:
Do I have to add the unit as a last parameter for that to work? If so, I think the diff would look similar :/ Or is there some other trick to make that work that I'm not aware of?

Reply:
Oh yes I didn't pay attention to that

Reply:
Would you still do that? Making the `default_answer` optional (defaulting to `Yes`) and adding the unit as positional parameter I mean?

Reply:
No I guess I would do like you did